addons/website/models/website.py

```diff
@@ -194,7 +194,9 @@ def is_menu_cache_disabled(self):
         Checks if the website menu contains a record like url.
         :return: True if the menu contains a record like url
         """
-        return any(self.env['website.menu'].browse(self._get_menu_ids()).filtered(lambda menu: re.search(r"[/](([^/=?&]+-)?[0-9]+)([/]|$)", menu.url)))
+        return any(self.env['website.menu'].browse(self._get_menu_ids()).filtered(
+            lambda menu: re.search(r"[/](([^/=?&]+-)?[0-9]+)([/]|$)", menu.url) or menu.group_ids
```

Comment:
Since the [commit that introduced this line](https://github.com/odoo/odoo/commit/d2a3438b59b6230e29e32342b2c1e8381e57cee6), there's a bug leading to an error 500 crash. I think we can take the opportunity of this PR to fix it. :)
Basically, the function checks `menu.url`, but this might return `None`, and that doesn't make `re.search()` happy.

You can reproduce it by taking one of the menus and removing its URL. (Real case scenario: create a toggle menu with submenus, where the toggle menu is not a link itself.)

Reply:
Ah but no, I didn't see it was a forward-port from 16.0. It should be fixed in its own commit.